zend_abstract_interface/call/tests/call.cc

```diff
@@ -0,0 +1,434 @@
+extern "C" {
+#include "call/call.h"
+#include "zai_sapi/zai_sapi.h"
+#include "zai_sapi/zai_sapi_extension.h"
+
+#include "zai_compat.h"
+}
+
+#include <catch2/catch.hpp>
+#include <cstdlib>
+#include <cstring>
+
+// clang-format off
+static inline bool zai_call_test_class(zend_class_entry **ce ZAI_TSRMLS_DC) {
+    zai_sapi_execute_script("./stubs/Stub.php");
+
+    zai_string_view scope =
+        ZAI_STRL_VIEW("ZendAbstractInterfaceCallStub");
+
+    *ce = zai_call_lookup_class(
+        ZAI_CALL_SCOPE_NAMED, &scope ZAI_TSRMLS_CC);
+
+    return *ce != NULL;
+}
+// clang-format on
+
+static inline bool zai_call_test_stub(zval **object ZAI_TSRMLS_DC) {
+    zend_class_entry *ce;
+
+    if (!zai_call_test_class(&ce ZAI_TSRMLS_CC)) {
+        return false;
+    }
+
+#if PHP_VERSION_ID < 70000
+    MAKE_STD_ZVAL(*object);
+#endif
+
+    return zai_call_new(*object, ce ZAI_TSRMLS_CC, 0);
+}
+
+#define TEST(name, code) TEST_CASE(name, "[zai_call]") { \
+        REQUIRE(zai_sapi_sinit()); \
+        REQUIRE(zai_sapi_minit()); \
+        REQUIRE(zai_sapi_rinit()); \
```

Comment:
Cannot we use `zai_sapi_spinup()` here? I guess we have this in our new PR as well